src/libsystemd/sd-event/event-util.c

```diff
@@ -163,7 +163,7 @@ int event_add_time_change(sd_event *e, sd_event_source **ret, sd_event_io_handle
 
 int event_add_child_pidref(
                 sd_event *e,
-                sd_event_source **ret,
+                sd_event_source **ret_event_source,
```

Comment:
Please use `ret`, as this provides only one result, and `event_source` is trivial by the type.

Reply:
Happy to drop that commit, but then we'll need to update all these argument comments to ret:

```
[jelle@carbon][~/projects/systemd]%grep bugprone-argument tidy.log | grep ret_event
../src/udev/udev-watch.c:633:36: error: argument name 'ret_event_source' in comment does not mat
ch parameter name 's' [bugprone-argument-comment,-warnings-as-errors]
../src/login/logind.c:99:43: error: argument name 'ret_event_source' in comment does not match p
arameter name 's' [bugprone-argument-comment,-warnings-as-errors]
```

Which seems undesired?

Reply:
Still I prefer `ret` here, even that requires to rename many comments.